pymc/tests/test_sampling.py

```diff
@@ -1450,9 +1450,14 @@ def test_step_args():
         a = pm.Normal("a")
         idata0 = pm.sample(target_accept=0.5, random_seed=1410)
         idata1 = pm.sample(nuts={"target_accept": 0.5}, random_seed=1410 * 2)
+        idata2 = pm.sample(target_accept=0.5, nuts={"max_treedepth": 10}, random_seed=1410)
+
+        with pytest.raises(ValueError):
```

Comment:
You can match the error string for a more precise test
```suggestion
        with pytest.raises(ValueError, match="..."):
```

Reply:
Thank you, I added a pattern here to improve the test.